src/db.c

```diff
@@ -8344,7 +8332,7 @@ void exp_trailstation(FILE *f, DataRow *p_station, int export_format) {
            fprintf(f,"<Placemark>");
            get_iso_datetime(p_station->sec_heard,timestring,True,True);
 
-           if (p_station->origin == NULL || p_station->origin[0] == '\0') { 
+           if (p_station->origin[0] == '\0') { 
```

Comment:
Why don't we need to check for NULL pointer anymore?

Reply:
Because it's not a pointer.  It's an array declared with a fixed dimension.  This test can never fail, and some compilers whine about it.